airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -1192,6 +1194,12 @@ def wait_for_batch(
 
         return result
 
+    def check_error_for_resource_is_not_ready_msg(self, error_msg: str) -> bool:
+        """Check that reason of error is resource is not ready."""
+        key_words = ["The resource", "is not ready"]
+        result = [word in error_msg for word in key_words]
+        return all(result) if result else False
```

Comment:
Can't we just `return all([word in error_msg for word in key_words])`  keeping the current logic, as we have static `key_words` we are always gonna have result from `[word in error_msg for word in key_words]` ?